manifests/a/ApheaBio/RcloneTray/1.0.0/ApheaBio.RcloneTray.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ApheaBio.RcloneTray
+PackageVersion: 1.0.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/aurang1655/ApheaBio/releases/download/1/RcloneTray_dist_RcloneTray_Setup_1.0.0.exe

```

Comment:
```
  InstallerUrl: https://github.com/aurang1655/ApheaBio/releases/download/1/RcloneTray_dist_RcloneTray_Setup_1.0.0.exe
```
Shouldn't the URL be https://github.com/dimitrov-adrian/RcloneTray/releases/download/v1.0.0/rclonetray-setup-1.0.0.exe here? With a hash of `b86f8b6123ce540934c89e1f629101ee04bae088c10f6e999fc58f97a1d2faeb`?